src/table/exportable/exports/csv.js

```diff
@@ -0,0 +1,48 @@
+export default function csv(data) {
+    if (!data) data = this.data.filtered[0].values.map(d => d.raw);
+
+    const CSVarray = [];
+
+    data.forEach((d, i) => {
+        //add headers to CSV array
+        if (i === 0) {
+            const headers = this.config.headers.map(header => `"${header.replace(/"/g, '""')}"`);
+            CSVarray.push(headers);
+        }
+
+        //add rows to CSV array
+        const row = this.config.cols.map(col => {
+            let value = d[col];
+
+            if (typeof value === 'string') value = value.replace(/"/g, '""');
+
+            return `"${value}"`;
+        });
+
+        CSVarray.push(row);
+    });
+
+    //transform CSV array into CSV string
+    const CSV = new Blob([CSVarray.join('\n')], { type: 'text/csv;charset=utf-8;' }),
+        fileName = `Selected Data.csv`,
```

Comment:
Let's avoid spaces in file names, and I tend to like filenames more descriptive. How about "webchartsTablesExport_yyyymmdd_hhmmss.csv".